pyopencl/invoker.py

```diff
@@ -138,7 +138,7 @@ def add_buf_arg(arg_idx, typechar, expr_str):
                 cl_arg_idx += 1
 
             if in_enqueue:
-                wait_for_parts .append(f"{arg_var}.events")
```

Comment:
Check that invoker cache is disabled by env var

Reply:
Done in #637